src/components/ha-sidebar.ts

```diff
@@ -772,9 +772,9 @@ class HaSidebar extends LitElement {
           border-bottom: 1px solid transparent;
           white-space: nowrap;
           font-weight: 400;
-          color: var(--primary-text-color);
+          color: var(--menu-text-color, --primary-text-color);
```

Comment:
Since this is just for the sidebar, I think the naming of the variable is wrong.

We already have `--sidebar-background-color` and `--sidebar-text-color`, you add name yours `--sidebar-menu-button-*`

Reply:
Sorry for late response (down on vacation) - although totally agree with you. Will change those names. :)